service/rtc/config.go

```diff
@@ -97,10 +124,8 @@ func (s ICEServers) IsValid() error {
 
 func (s ICEServers) getSTUN() string {
 	for _, cfg := range s {
-		for _, u := range cfg.URLs {
-			if strings.HasPrefix(u, "stun:") {
-				return u
-			}
+		if cfg.IsSTUN() && len(cfg.URLs) > 0 {
+			return cfg.URLs[0]
```

Comment:
This might mean: if there's an incorrect first url but a correct second url, then we just returned the incorrect one?